modules/ROOT/pages/dw-arrays.adoc

```diff
@@ -16,11 +16,10 @@ DataWeave script.
 specified number of elements.
 | xref:dw-arrays-functions-drop.adoc[drop] | Drops the first `n` elements. It returns the original array when `n &lt;= 0`
 and an empty array when `n > sizeOf(array)`.
-| xref:dw-arrays-functions-dropwhile.adoc[dropWhile] | Drops elements from the array while the condition is met but stops the selection process
+| xref:dw-arrays-functions-dropwhile.adoc[dropWhile] | Drops elements from the array while the condition is met but stops the selection process 
 when it reaches an element that fails to satisfy the condition.
 | xref:dw-arrays-functions-every.adoc[every] | Returns `true` if every element in the array matches the condition.
-| xref:dw-arrays-functions-firstwith.adoc[firstWith] | Returns the first element that satisfies the condition, or returns `null` if no
-element meets the condition.
+| xref:dw-arrays-functions-firstwith.adoc[firstWith] | Returns the first element that satisfy the condition or null y non element match.
```

Comment:
TODO: Should be
"This example shows how `firstWith` behaves when an element matches and when an element does not match."